### DISCUSSION TITLE
Different slide with different height

### DISCUSSION THREAD

**Discussion by lawchihon** [nolimits4web/swiper#4121] [Q&A]
This is a (multiple allowed):
* [ ] bug
* [x] enhancement
* [ ] feature-discussion (RFC)

* Swiper Version: 5.
* Platform/Target and Browser Versions: IOS
* Live Link or JSFiddle/Codepen or website with isssue: https://codepen.io/lawchihon/pen/ZEpYbra

### What you did
EXPLAIN WHAT YOU DID, PREFERABLY WITH CODE EXAMPLES, HERE.
I am trying to set different height on different slide.
Tried to set height as auto or fixed pixel, but either the style get overwritten or behaviour become weird

### Expected Behavior
Allow different height on each slide

### Actual Behavior
Forced to have all slide to have same height

**Comment by ygj6** [comment]
@lawchihon Hi, the following is the author’s opinion, Is this what you want?
> If you need different slide widths/heights this is exactly what slidesPerView: 'auto' mode exist, where it respects slide sizes and doesn't override anything

**Comment by andreaslarssen** [comment]
@ygj6 This seems to have no effect on slide individual height
